btree/btree.c

```diff
@@ -1885,12 +1869,45 @@ static int64_t mem_tree_get(struct node_op *op, struct segaddr *addr, int nxt);
  */
 static int64_t tree_get(struct node_op *op, struct segaddr *addr, int nxt)
 {
-	int        nxt_state;
+	struct td              *tree = NULL;
+	struct nd              *node = NULL;
+
+	if (addr != NULL) {
+		nxt  = node_get(op, NULL, addr, nxt);
+		node = op->no_node;
+		if (node != NULL) {
```

Comment:
In which condition node can be NULL?

Reply:
Hmm. I realise that it wont be NULL.
I will remove this check.